lib/wibox/widget/graph.lua

```diff
@@ -21,7 +21,6 @@ local setmetatable = setmetatable
 local ipairs = ipairs
 local math = math
 local table = table
-local type = type
```

Comment:
This removal isn't there in my local version of the commit and `type` is used in the file. You must have saved an old version of my commit or something.

Reply:
I first pushed 1 test commit to the branch before making the force push. Given it worked, it means the version I had locally was still the head of the PR at that point. I removed `type` because `luacheck` said it was unused and made the build fail. Maybe we have different versions of luacheck?

Reply:
Sorry, I have missed these comments of yours, because my dumb overlarge force-pushes hid these comments in "94 hidden items, Load more..." and I somehow didn't get a notification.

>I removed `type`

Well this is weird, because the removal somehow ended up in one of my commits. I saw the change in the diff between our branches, but couldn't find the change in any of your commits, which puzzled me immensely, so I had to git-blame it and found it in my own very second commit, in which I vaguely remembered doing the removal, before deciding against it, but it was a long time ago, so I suspected that you saved my old commit. 

I don't know, if the problem is in luacheck (I have 0.23.0), but I do use `type` in
```lua
    if not group_start or type(group_start) == "number" then
```
so it's weird that luacheck would have complained about it, given that we even have tests that exercise this line.

Reply:
Oh wait, you don't tell me... could it be that every commit of a PR must pass checks on its own and not only the tip? That would explain what you've done. I'm sorry then.

The `type` variable indeed gets unused for a while, but it should be reintroduced later. I'll do that then.